app/controllers/invitations_controller.rb

```diff
@@ -42,7 +42,7 @@ def update
     if params[:refuse]
       @invitation.refuse
       flash[:info] = "You have refused this invitation."
-      redirect_to :back
+      redirect_to root_url
```

Comment:
I changed this as it seemed better to leave the proposal page if they had refused the invite.